docs/user/tutorial/quickstart/index.rst

```diff
@@ -13,7 +13,6 @@ This tutorial is available for:
    netbeans
    intellij
    maven
-   java11
```

Comment:
The contents of the java 11 tutorial (being module based) is still valuable and should be incorporated in the other tutorials rather than lost.

Reply:
Please do, if you find anything of value in there (modules have been the norm for years, not sure we need to say anything interesting about them). The parts about developing in a newer JDK targetting 11 I would remove or discourage, it's just a way to get caught in API changes and fail the PR builds.... In both Linux and Windows it's easy to have mulitple JDKs installed, so one developing on GT should just have Java 11 handy. Maybe it's hard on OSX? Seems unlikely but not impossible.

Reply:
I guess I mean that the Quickstarts should have a section on how to run on module path.

Reply:
The text of the module section has been added back in its own separate chapter. If that's not enough could you please add whatever is the missing bit?

Reply:
I've checked again the old ``java11.rst``, I don't see any special bit on how to "run on module path"